pkgs/top-level/python-packages.nix

```diff
@@ -11114,7 +11114,7 @@ self: super: with self; {
 
   rawkit = callPackage ../development/python-modules/rawkit { };
 
-  ray = callPackage ../development/python-modules/ray { redis = pkgs.redis; };
```

Comment:
Are you sure `redis` is supposed to be the Python library?

Reply:
Actually, it's not used internally.

Reply:
The Python library isn't used or redis isn't used?

Reply:
Ray is a distributed framework for Python, and they used to use redis for synchronization across machine, now they are moving away from redis.
Since commit c6678ba42d64aa41d7da738b274138b86dd75a9b, redis was removed from it's dependency.
So yes, redis is currently not needed for this library.